server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreRestoreIT.java

```diff
@@ -91,6 +92,7 @@ public void testRTSRestoreWithNoDataPostCommitPrimaryReplicaDown() throws Except
      * Simulates all data restored using Remote Translog Store.
      * @throws IOException IO Exception.
      */
+    @TestIssueLogging(value = "_root:TRACE", issueUrl = "https://github.com/elastic/elasticsearch/issues/11085")
```

Comment:
Do we need TRACE for all the classes? If possible, please limit it to specific class.

Reply:
not completely sure if adding trace to certain classes only will help but if this seems too much i can start with some classes as trace which i suspect and enable debug at root level

Reply:
Sorry, the link here is to the wrong repo, that's what I meant.

Should be https://github.com/opensearch-project/OpenSearch/issues/11085